packages/website/src/SideNavigation.tsx

```diff
@@ -88,6 +88,7 @@ export const Navigation: React.FunctionComponent = () => {
                 <NavLink href="/layout/BorderedLine" label="Bordered line" />
                 <NavLink href="/layout/BrowserPreview" label="Browser preview" />
                 <NavLink href="/layout/Chart" label="Chart" />
+                <NavLink href="/layout/ChildForm" label="Child form" />
```

Comment:
you forgot to change it in the home page :eagle:

Reply:
also would it be overkill to patante something to make the homepage build automatically :see_no_evil:

Reply:
from the home page, breaks because the component is not there's anymore... maybe we should also think of a better 404 page? :eyes: 
![image](https://user-images.githubusercontent.com/63734941/158796866-77cb59de-91ed-4cbc-90f6-f4fc6f35c1f8.png)